src/Sulu/Bundle/ContactBundle/Content/Types/SingleAccountSelection.php

```diff
@@ -11,14 +11,15 @@
 
 namespace Sulu\Bundle\ContactBundle\Content\Types;
 
+use PHPCR\NodeInterface;
 use Sulu\Bundle\ContactBundle\Api\Account;
 use Sulu\Bundle\ContactBundle\Contact\AccountManager;
 use Sulu\Bundle\WebsiteBundle\ReferenceStore\ReferenceStoreInterface;
 use Sulu\Component\Content\Compat\PropertyInterface;
+use Sulu\Component\Content\ComplexContentType;
 use Sulu\Component\Content\PreResolvableContentTypeInterface;
-use Sulu\Component\Content\SimpleContentType;
 
-class SingleAccountSelection extends SimpleContentType implements PreResolvableContentTypeInterface
+class SingleAccountSelection extends ComplexContentType implements PreResolvableContentTypeInterface
```

Comment:
Is this not the same as for the `SingleContactSelection` think there we need to change it also?

Reply:
I tested both field types, and the `single_contact_selection` was working, so I didn't touch it. But IIRC for the contacts it's not an `AutoComplete`, so that's probably the difference.

Reply:
@danrot ok that make sense.